src/library/dao/trackdao.h

```diff
@@ -53,16 +53,15 @@ class TrackDAO : public QObject, public virtual DAO, public virtual GlobalTrackC
 
     TrackId getTrackIdByRef(
             const TrackRef& trackRef) const;
-    QList<TrackId> getAllTrackIds(
+    QList<TrackRef> getAllTrackRefs(
             const QDir& rootDir);
 
     TrackPointer getTrackByRef(
             const TrackRef& trackRef) const;
 
     // Returns a set of all track locations in the library.
-    QSet<QString> getTrackLocations();
+    QSet<QString> getAllTrackLocations();
     QString getTrackLocation(TrackId trackId);
-    QStringList getTrackLocations(const QList<TrackId>& trackIds);
```

Comment:
Agree with removing this method - it seems out of place to have an isolated list of locations for specific tracks without knowing the mapping between them!